packages/colorpicker/lib/colorpicker.dart

```diff
@@ -0,0 +1,116 @@
+library colorpicker;
+
+import 'package:colorpicker/constants/colors.dart';
```

Comment:
Comment above results in these 3 imports being just one import of the /lib colorpicker